tests/src/Functional/Thunder2UpdateTest.php

```diff
@@ -19,6 +19,17 @@
 class Thunder2UpdateTest extends UpdatePathTestBase {
   use ThunderAwsTestFixtureTrait;
 
+  /**
+   * {@inheritdoc}
+   */
+  protected function setUp() {
+    parent::setUp();
+
+    // Remove fb_instant_articles 1.x broken config..
+    $configFactory = \Drupal::configFactory();
+    $configFactory->getEditable('views.view.fb_instant_articles')->delete();
```

Comment:
Item 1: Appearantly this image style is used in metatag.metatag_defaults.node__article.yml

Item 2: Removed dependency from distribution (since its already set in thunder_fia)

Item 3: Using strict version in thunder_fia 2.x now